models/reviews_model.js

```diff
@@ -16,6 +16,20 @@ exports.selectReview = (review_id) => {
     })
 }
 
+exports.selectComments = (review_id) => {
+    return db.query('SELECT * FROM comments WHERE review_id = $1', [review_id])
+    .then(({rows}) => {
+        if(rows.length === 0) {
+            return Promise.reject({
+                status: 404,
+                message: 'Not Found'
+            })
```

Comment:
Sadly, a simple length check will not suffice to find out whether the review exists but just has no comments